src/main/java/bio/terra/app/controller/SnapshotsApiController.java

```diff
@@ -265,6 +267,29 @@ public ResponseEntity<FileModel> lookupSnapshotFileByPath(
     return ResponseEntity.ok(fileModel);
   }
 
+  @Override
+  public ResponseEntity<SnapshotPreviewModel> querySnapshotDataById(
+      UUID id, String table, QueryDataRequestModel queryDataRequest) {
+    logger.debug("Verifying user access");
```

Comment:
This looks like a leftover from testing!

Reply:
I'll clean these up!